### DISCUSSION TITLE
wiki/help-pages/launcher-settings/

### DISCUSSION THREAD

**Discussion by giscus** [PrismLauncher/prismlauncher.org#667] [Wiki]
# wiki/help-pages/launcher-settings/

An Open Source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability.

https://prismlauncher.org/wiki/help-pages/launcher-settings/

**Comment by Chancers315** [comment]
How do I reset the instances folders to where it used to be since it was taking up a whole bunch of space and I changed it to my :D drive and now it's lost :(

**Reply by vermetra** [reply]
Alright brochacho i know im very late, but go to the launcher settings and where it says instances click browse and that should allow you to select the old folder.
